CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 0.10.0
```

Comment:
I think you can pretty much declare 1.0.0 , as we for all purposes we are treating this gem as semver, no point of staying in 0.x versions